workflows/tests/test_assemble_study_flow.py

```diff
@@ -184,7 +211,14 @@ def suspend_side_effect(wait_for_input=None):
 
     assert (
         analyses.models.Assembly.objects.filter(status__assembly_completed=True).count()
-        == 1
+        == 2
+    )
+    # for SE assembler should be swapped to megahit
+    assert (
+        analyses.models.Assembly.objects.filter(assembler__name='megahit').count() == 1
+    )
+    assert (
+        analyses.models.Assembly.objects.filter(assembler__name='metaspades').count() == 2
```

Comment:
```suggestion
    assert (
        analyses.models.Assembly.objects.filter(assembler__name__iexact='megahit').count() == 1
    )
    assert (
        analyses.models.Assembly.objects.filter(assembler__name__iexact='metaspades').count() == 2
```

I think the mixed of the names casing may be causing trouble on this